app/docs/0.13.x/proxy.md

```diff
@@ -1074,6 +1074,26 @@ standard HTTP proxy.
 
 [Back to TOC](#table-of-contents)
 
+### WebSocket and TLS
+
+When setting up the Service to point to your upstream WebSocket service you
+should pick the right protocol you want to use between Kong and the upstream.
+
+If you want to use TLS (`wss`) then the upstream WebSocket service must be
+defined using the `https` protocol and the matching port (usually 443). To
+connect without TLS (`ws`) the `http` protocol and port (usually 80) should be
+used.
+
+Similar to regular requests Kong will accept `ws` and `wss` connections on its
+respective `http` and `https` ports. To restrict clients to only accessing over
+TLS, set the `protocols` property of the Route to `https` only.
+
+This is independent of the connection setting for the upstream. So if you want
+Kong to terminate SSL, you can accept `wss` only from client, but proxy to the
```

Comment:
We've been (correctly) mentioning TLS all along this paragraph and suddenly we mention SSL. Better stay consistent.

Reply:
I actually looked this up. The common terminology is "SSL termination" (several occurrences in the rest of this doc as well). As I found it weird to mention "TLS termination", I stuck with SSL.

lmk if you want me to change it anyway.

Reply:
maybe "terminate SSL/TLS" for the non-tech-savvy reader?